packages/theme/components/BottomNavigation.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+<!-- TODO: create logic with isActive prop for BottomNavigationItems -->
```

Comment:
Could you please create github issues for implementing these TODOs? :)

Reply:
done